docker/ngc_overview_hugectr.md

```diff
@@ -0,0 +1,101 @@
+## What is Merlin for Recommender Systems?
+
+NVIDIA Merlin is a framework for accelerating the entire recommender systems pipeline on the GPU: from data ingestion and training to deployment. Merlin empowers data scientists, machine learning engineers, and researchers to build high-performing recommenders at scale. Merlin includes tools that democratize building deep learning recommenders by addressing common ETL, training, and inference challenges. Each stage of the Merlin pipeline is optimized to support hundreds of terabytes of data, all accessible through easy-to-use APIs. With Merlin, better predictions than traditional methods and increased click-through rates are within reach.
+
+The Merlin ecosystem has four main components: Merlin ETL, Merlin Dataloaders and Training, and Merlin Inference.
+
+## Merlin HugeCTR
+
+The merlin-hugectr container allows users to do preprocessing and feature engineering with NVTabular, and then train a deep-learning based recommender system model with HugeCTR, and serve the trained model on Triton Inference Server.
+
+As the ETL component of the Merlin ecosystem, NVTabular is a feature engineering and preprocessing library for tabular data designed to quickly and easily manipulate terabyte scale datasets used to train deep learning based recommender systems. The core features are explained in the API documentation and additional information can be found in the GitHub repository.
```

Comment:
* NVTabular performs data preprocessing and feature engineering for tabular data.  The library can operate on small and large datasets--scaling to manipulate terabyte-scale datasets that are used to train deep learning recommender systems.

* HugeCTR can train deep learning recommender models and is written in CUDA C++ to provide optimal performance with NVIDIA GPUs.  The library is a recommender-specific framework with optimized data loaders that can perform distributed training across multiple GPUs and nodes. HugeCTR provides strategies for scaling large embedding tables beyond available memory. (I snipped the CTR part because I think Karl and Even thought that was limiting while we developed the Merlin repo README. This proposed second bullet replaces the "HugeCTR, a dedicated..." para.)

(FYI, let's not make any text into links to the repo in this section.  Let the reader concentrate on understanding what is in the container rather than contemplate whether "better' information is behind the link. Links belong at the bottom of the page for curious readers.)